cmd/vaults/commands.go

```diff
@@ -156,7 +133,8 @@ func newVaultCreateCommand() *cli.Command {
 }
 
 func newStreamCommand() *cli.Command {
-	var privateKey string
+	var privateKey, dburi, tables string
+	var winSize int64
```

Comment:
should `winSize` be configurable per table?

Reply:
do you think there's a good reason for it? we could, but we would need an export gouroutine per table which sounds more complicated.

Reply:
agreed—i think a single window size makes sense as a first feature. when i stream multiple tables, I'm essentially just trying to replicate the full db, so one window size is sufficient.

multiple windows _could_ be useful, i suppose, but that's something we can wait on to hear directly from a dev. personally, i don't think i'd need it unless i was trying to really optimize for something.